src/domain/auth/state/AuthenticationUtils.ts

```diff
@@ -0,0 +1,29 @@
+import { store, persistor } from '../../app/state/AppStore';
+import { resetFormValues } from '../../registration/state/RegistrationActions';
+import { clearProfile } from '../../profile/state/ProfileActions';
+import { resetBackendAuthentication } from './BackendAuthenticationActions';
+import client from '../../api/client';
+import { resetUIState } from '../../app/state/ui/UIActions';
+
+type FlushStateOptions = {
+  keepUserFormData?: boolean;
+};
+
+export const flushAllState = (options: FlushStateOptions) => {
```

Comment:
Would be great with a comment here about logging out so that people can find it if they search for logout or logOut or log out in our codebase

Reply:
No i dont include log out in this function, so no need to explain here